apsuite/commisslib/measure_tune_bpm.py

```diff
@@ -0,0 +1,283 @@
+"Methods for measure tunes by bpm data"
+
+import numpy as _np
+import time as _time
+import pyaccel as _pa
+from numpy.fft import rfft as _rfft,  rfftfreq as _rfftfreq
+import matplotlib.pyplot as _plt
+
+from siriuspy.sofb.csdev import SOFBFactory
+
+from siriuspy.devices import CurrInfoBO, \
+    Trigger, Event, EVG, RFGen, FamBPMs
+
+from apsuite.utils import ParamsBaseClass as _ParamsBaseClass, \
+    ThreadedMeasBaseClass as _ThreadBaseClass
+
+
+class BPMeasureParams(_ParamsBaseClass):
+    """."""
+    def __init__(self):
+        """."""
+        super().__init__()
+        self.nr_points_after = 0
+        self.nr_points_before = 10000
+        self.bpms_timeout = 30  # [s]
+        self.trigger_source = 'DigBO'
+        self.trigger_source_mode = 'Injection'
+        self.extra_delay = 0
+        self.nr_pulses = 1
+
+
+class BPMeasure(_ThreadBaseClass):
```

Comment:
I don't think the name of this class is appropriated. It does not give a hint of what this class is about. I think this is a good point to change the name of the classes before merging into master. This applies to BPMeasure, BPMeasureParams and the file name `measure_tune_bpm.py`.

Reply:
Renamed, thanks!